kroxylicious-systemtests/src/test/java/io/kroxylicious/systemtests/extensions/TestKubeKmsFacadeInvocationContextProvider.java

```diff
@@ -36,7 +36,6 @@ public boolean supportsTestTemplate(ExtensionContext context) {
     @Override
     public Stream<TestTemplateInvocationContext> provideTestTemplateInvocationContexts(ExtensionContext context) {
         return TestKmsFacadeFactory.getTestKmsFacadeFactories()
-                .filter(x -> x.getClass().getName().contains("systemtests"))
```

Comment:
I think this filter was ensuring that the System Tests ignore test facades such as InMemoryTestKmsFacade or the VaultTestKmsFacade.  what will stop the system tests running these?